src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/NewHandlerTest.java

```diff
@@ -36,14 +36,14 @@
     @Override
     protected String getPath(String filename) throws IOException {
         return super.getPath("checks" + File.separator
-                + "indentation" + File.separator + filename);
+                + "indentation" + File.separator + "newhandler" + File.separator + filename);
     }
 
     @Test
     public void testInvalidLabel() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(IndentationCheck.class);
```

Comment:
@romani Why do we have this single test not even testing the specific class it is named after?
Having this test indentation outside of `IndentationCheckTest` means it is bypassing all the extra checks we added to the `Indentation` tests.
IMO, we should delete this class and move the only method to `IndentationCheckTest`.

I'm also not sure why this test is called `label` as input has nothing to do with `label:`.